4-Shaders-MoreAttributes/Window.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using OpenTK;
+using OpenTK.Graphics;
+using OpenTK.Graphics.OpenGL4;
+using OpenTK.Input;
+using LearnOpenTK.Common;
+
+namespace LearnOpenTK
+{
+    // In this project we will be assigning 3 colors to the triangle, one for vertex, but the output will be a mixture
+    // Of the three depending on how far away from the assignment point the color pixel is
+    // This comes down to something called fragment interpolation in the fragment shader
+    // When rendering a triangle the rasterization stage usually results in a lot more fragments than vertices originally specified. 
+    // The rasterizer then determines the positions of each of those fragments based on where they reside on the triangle shape
+    // Based on these positions, it interpolates all the fragment shader's input variables
```

Comment:
You use a lot of fancy words in this bit of text which are not necessary especially not for someone trying to wrap their head around shaders and opengl for the first time. Try explaining it simpler and with fewer sentences. (also some of the stuff is new to me although it makes sense. Just goes to show how little you actually need to know it tho.)